src/Compilers/Server/CommandLine/VBCSCompilerCommandLine.projitems

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(RepositoryEngineeringDir)targets\GenerateCompilerExecutableBindingRedirects.targets" />
+  <PropertyGroup>
```

Comment:
Not related to this PR: the C# sibling of this file has `    <Prefer32Bit>false</Prefer32Bit>`. Should we have that here as well?

Reply:
Believe that only overrides the default when hitting F5. Essentially has no product impact. But yes it should be consistent between the exes.